meeting-minutes/team-minutes/nov16-team-minutes.md

```diff
@@ -15,10 +15,36 @@
 - 「Aleks」Do we want to create a logout page for the website?
 - 「Aleks」When selecting different buttons from mentor home the top bar switched to log in, not sure how to fix this.
 - 「Aleks」We can add a task to create the UI for the mentor home page (right now it is empty and only used for buttons) 
-
+- 
 
 ## Detailed Notes:
+- Everyone now has access to DB
+- Be careful when you are going through DB. try not to delete it, etc.
+- Passwords are not stored in DB
+- Authentication and real time are not linked (only if someone manually links them through code)
+- Creates user in firebase by passing in email/password to firebase, but does not store password
+- Kat will be the "leader" for peer testing
+- re-work the refine UI issue (#246)
+- team meeting on the 18th at 7am PST (one hour long)
 - 
 
 ## Action Items (Due by next sprint: November 23rd):
+- Ask more about video demo that is due on 25th, get more information maybe email prof if needed
+- Kat will be the "leader" for peer testing, she will give us roles/what to do/give us more information about peer testing
+- delete pending tab
+- only keep mentorhome, accepted, and suggested tab
+- kat will do the issue 269, description + menotr and mentee forms and skills fiels to mentors form

```

Comment:
Just a slight modification: Eric will be working on the description/skill fields

Reply:
Updated! thanks for letting me know